test/test_udp_socket.rs

```diff
@@ -49,13 +66,18 @@ fn test_send_recv_udp(tx: UdpSocket, rx: UdpSocket, connected: bool) {
 
     // ensure that the sockets are non-blocking
     let mut buf = [0; 128];
-    assert_eq!(ErrorKind::WouldBlock, rx.recv_from(&mut buf).unwrap_err().kind());
+    assert_eq!(
```

Comment:
Can you undo all these rustfmt changes to reduce the size of the diff?

Reply:
Ok, I will do that.